ingenialink/ethercat/servo.py

```diff
@@ -320,6 +323,49 @@ def set_pdo_watchdog_time(self, timeout: float) -> None:
             self.ETHERCAT_PDO_WATCHDOG, self.SECONDS_TO_MS_CONVERSION_FACTOR * timeout
         )
 
+    def read_eeprom(
+        self,
+        address: int,
+        length: int = DEFAULT_EEPROM_READ_BYTES_LENGTH,
+        timeout: int = DEFAULT_EEPROM_OPERATION_TIMEOUT_uS,
+    ) -> bytes:
+        """Read from the ESC EEPROM.
+
+        Args:
+            address: EEPROM address to be read.
+            length: Length of data to be read. By default, 2 bytes are read.
+            timeout: Operation timeout (microseconds). By default, 200.000 us.
+
+        Returns:
+            EEPROM data. The read data.
+
+        """
+        data = bytes()
+        start_address = address
+        while len(data) < length:
+            data += self.slave.eeprom_read(start_address, timeout)
+            start_address += 2
+        if len(data) > length:
+            data = data[:length]
+        return data
+
+    def write_eeprom(
+        self, address: int, data: bytes, timeout: int = DEFAULT_EEPROM_OPERATION_TIMEOUT_uS
+    ) -> None:
+        """Write to the ESC EEPROM.
+
+        Args:
+            address: EEPROM address to be written.
+            data: Data to be written. The data length must be a multiple of 2 bytes.
```

Comment:
Should we check that len(data) % 2 == 0?

Reply:
It is already checked by [PySOEM](https://github.com/bnjmnp/pysoem/blob/master/pysoem/pysoem.pyx#L821), but I could add it as well.